### ISSUE TITLE
Filtered Training Dataset Size And Training Time

### ISSUE THREAD

**Issue by LianShuQuan** [Yan98/Grounding-R1#2]
Good Work！
After cleaning using omniparser，there still are 1462321 examples，is that right?
As you metioned “The training process can be completed around in approximately 800 H100 GPU-hours over 250 optimization iterations.”, how did you train in so many examples with only 800 H100 GPU-hours ? Because you used very big batch size?

**Comment by Yan98** [comment]
Thank you for your interest. We do not use the full dataset for training; instead, a subset is randomly sampled during the process. Based on our empirical observations, you can assess model convergence by monitoring the standard deviation of the reward. Usually, when the reward's standard deviation consistently stays below 0.05, it indicates that training can be stopped.
